src/components/Text/index.tsx

```diff
@@ -15,21 +14,20 @@ export interface TextProps {
 }
 
 const CustomText = styled.Text`
-  color: ${(props: { color: string; theme: ThemeAttributes }) =>
-    props.color === 'primary'
-      ? props.theme.PRIMARY_TEXT_COLOR
-      : props.theme.SECONDARY_TEXT_COLOR};
+  color: ${(props: { color: TextProps['color']; theme: ThemeAttributes }) =>
+    props.color && `${props.theme[props.color]}`};
   font-size: ${(props: { size: string }) => `${props.size}px`};
   font-family: ${(props: { font: string }) => `${props.font}`};
   max-width: ${(props: { maxWidth: number }) =>
     props.maxWidth ? `${props.maxWidth}px` : '100%'};
   text-align: ${(props: { textAlign: string }) => `${props.textAlign}`};
-  text-decoration: ${(props: { textDecoration: string }) => `${props.textDecoration}`};;
+  text-decoration: ${(props: { textDecoration: string }) =>
+    `${props.textDecoration}`}; ;
```

Comment:
Warning, there are two semicolons

Reply:
It happened when I rebase main
I fix it right away